### ISSUE TITLE
ValueError : You are trying to load a weight file containing 3 layers into a model with 0 layers

### ISSUE THREAD

**Issue by menkaur** [SciSharp/Keras.NET#50]
I'm loading model trained with following:

```
model = keras.models.Sequential([
    keras.layers.Flatten(input_shape=[HISTORY_LEN]),
    keras.layers.Dense(PREDICTION_LEN),
    keras.layers.LeakyReLU(),
    keras.layers.Dense(PREDICTION_LEN),
    keras.layers.LeakyReLU(),
    keras.layers.Dense(PREDICTION_LEN)
])

model.compile(loss="mse", optimizer="adam")
history = model.fit(x, y, epochs=200,validation_data=(x_v,y_v))
```

Here's code I'm using:
             
  ```
 var model = new Keras.Models.Sequential();
                model.Add(new Flatten());
                model.Add(new Dense(60));
                model.Add(new LeakyReLU());
                model.Add(new Dense(60));
                model.Add(new LeakyReLU());
                model.Add(new Dense(60));

                model.Compile(optimizer: "adam", loss: "mse");

                model.LoadWeight(@"D:\training-data\models\1\2.2_5d-1h.3layers.e2_23e-6.weights.h5");
```
This fails with following exception: 

> Python.Runtime.PythonException: 'ValueError : You are trying to load a weight file containing 3 layers into a model with 0 layers.'

**Comment by deepakkumar1984** [comment]
Can you save the model to json using model.save() and load that? Just to see if the error with load weight

**Comment by menkaur** [comment]
Yes, when saved and loaded in one file (model+weights), the model loads without exceptions

Here is the content of the json file. I entered model configuration manually because there was an issue with loading this file as well

`
{"class_name": "Sequential", "config": {"name": "sequential", "layers": [{"class_name": "Flatten", "config": {"name": "flatten", "trainable": true, "batch_input_shape": [null, 3600], "dtype": "float32", "data_format": "channels_last"}}, {"class_name": "Dense", "config": {"name": "dense", "trainable": true, "dtype": "float32", "units": 60, "activation": "linear", "use_bias": true, "kernel_initializer": {"class_name": "GlorotUniform", "config": {"seed": null}}, "bias_initializer": {"class_name": "Zeros", "config": {}}, "kernel_regularizer": null, "bias_regularizer": null, "activity_regularizer": null, "kernel_constraint": null, "bias_constraint": null}}, {"class_name": "LeakyReLU", "config": {"name": "leaky_re_lu", "trainable": true, "dtype": "float32", "alpha": 0.30000001192092896}}, {"class_name": "Dense", "config": {"name": "dense_1", "trainable": true, "dtype": "float32", "units": 60, "activation": "linear", "use_bias": true, "kernel_initializer": {"class_name": "GlorotUniform", "config": {"seed": null}}, "bias_initializer": {"class_name": "Zeros", "config": {}}, "kernel_regularizer": null, "bias_regularizer": null, "activity_regularizer": null, "kernel_constraint": null, "bias_constraint": null}}, {"class_name": "LeakyReLU", "config": {"name": "leaky_re_lu_1", "trainable": true, "dtype": "float32", "alpha": 0.30000001192092896}}, {"class_name": "Dense", "config": {"name": "dense_2", "trainable": true, "dtype": "float32", "units": 60, "activation": "linear", "use_bias": true, "kernel_initializer": {"class_name": "GlorotUniform", "config": {"seed": null}}, "bias_initializer": {"class_name": "Zeros", "config": {}}, "kernel_regularizer": null, "bias_regularizer": null, "activity_regularizer": null, "kernel_constraint": null, "bias_constraint": null}}]}, "keras_version": "2.2.4-tf", "backend": "tensorflow"}
`

**Comment by dsimeonov140** [comment]
I have the same issue. And when I save model to one file (model+weights), I can load it but Sequential.LoadModel return BaseModel type and I can't convert it to Sequential, so I did it that way:

Save model to one file (model+weights):
`model.Save(filename);`

When you need to load it do the following:
```
var model = new Keras.Models.Sequential();
              model.Add(new Flatten());
              model.Add(new Dense(60));
              model.Add(new LeakyReLU());
              model.Add(new Dense(60));
              model.Add(new LeakyReLU());
              model.Add(new Dense(60));

              model.Compile(optimizer: "adam", loss: "mse");

var model_tmp = Sequential.LoadModel(filename);
// I don't now why, but you must call model.Evaluate before calling model.SetWeights
model.SetWeights(model_tmp.GetWeights());
```

I think this error is not in Keras.NET but in keras.
When I am using only Dense layers there is no problem to load weights, but if I have LSTM or GRU layers this error is thrown.

**Comment by deepakkumar1984** [comment]
Closing this issue as this is keras python issue. Please raise with the Keras team
